promote/README.md

```diff
@@ -1,12 +0,0 @@
-# Thin Client Promotion
```

Comment:
For clarification reasons (since this repository is open), I think we should make it clear, that even though we don't promote the squasfs anymore automatically, the docker image is still bootable once you convert it to squasfs locally.